jobs/build/update-3.x-base-images/build.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+#➜  ~ docker run -it brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/jboss/openjdk18-rhel7:latest id
+#uid=185(jboss) gid=185(jboss) groups=185(jboss)
+#➜  ~ docker run -it  brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/elasticsearch:latest id
+#uid=1000(elasticsearch) gid=1000(elasticsearch) groups=1000(elasticsearch)
+#➜  ~  docker run -it brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/ansible-runner:latest  id
+#uid=0(root) gid=0(root) groups=0(root)
+# `rhel7` is root 0`
+# `nodejs6` is 1001`
+
+build_common() {
+    TAG="rhaos-4.0-rhel-7"
+    URL="http://pkgs.devel.redhat.com/cgit/containers/openshift-enterprise-base/plain/.oit/signed.repo?h="
+    USER_USERNAME="--user=ocp-build"
+    REPO=${URL}+${TAG}
+    TARGET_DIR=build-$1
+    rm -rf ${TARGET_DIR}
+    rhpkg ${USER_USERNAME} clone --branch ${TAG} containers/openshift-enterprise-base ${TARGET_DIR}
```

Comment:
this only really needs to happen once. you can reuse the clone.
as written it will actually try to create a clone within the clone each time (rhpkg might refuse, don't remember for sure)